test/Swagger.ObjectModel.Tests/SwaggerPropertyConventionTests.cs

```diff
@@ -21,7 +21,6 @@ public class SwaggerPropertyConventionTests
         /// This allows us to rename properties and enum values without worrying about breaking the Swagger JSON schema.
         /// </summary>
         /// <param name="member">The member to test.</param>
-        [Theory]
```

Comment:
Removed this because xunit was correctly flagging that this test had no inputs to run through. Just disabled it for now

Reply:
Maybe put a TODO or something to fix it later?

Reply:
Eh, I'm partially not a huge fan of TODOs (would rather create issues where possible). Besides I'll probably be back here soon, unless I'm beaten to it :P